### PR TITLE
Tenant's Farm

### PR DESCRIPTION
**Pull request by terrapin47**

### CODE REVIEW

**Comment by terrapin47** [comment]
@imcgames, please check your email.  It's important.

File: ITEM.tsv [hunk 1]
```diff
@@ -6842,7 +6842,7 @@ ITEM_20150918_006841	Farm equipment for making furrows. It has a sharp point.
 ITEM_20150918_006842	Farm equipment for removing stones. It is well made.
 ITEM_20150918_006843	Farm equipment for separating fields. It has a lot of sharp points.
 ITEM_20150918_006844	The letter Horacious wrote for borrowing the farm equipment. Readable.
-ITEM_20150918_006845	A Neolle pulled out to its rots. The root and leaf are used for consumption.
+ITEM_20150918_006845	A Niole pulled out to its roots. The root and leaf are used for consumption.
 ITEM_20150918_006846	A magic stone made by Yonas. They say it can be used to destroy the magic circle.
```
Comment (Sunaries) [inline ITEM.tsv]: Yonas -> Yoana

**Comment by Sunaries** [comment]
Good to merge.

**Comment by imcgames** [comment]
:+1:

**Pull request merged**